dashboard/logs/load.log

```diff
@@ -0,0 +1,24 @@
+2024-08-08 11:54:46,074 - logs - INFO - ---> Logging initiated.
```

Comment:
So we're still loading log files. I think maybe your .gitignore is behind.

Not an issue for now, just makes the repo a bit bloated.